### DISCUSSION TITLE
Should we release 0.70.1?

### DISCUSSION THREAD

**Discussion by dmytrorykun** [reactwg/react-native-releases#32] [Patches]
Released, see https://github.com/reactwg/react-native-releases/discussions/34
---

## Should we release 0.70.1?

Current Release: 0.70.0

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### Issues we are aware of

1. https://github.com/facebook/react-native/issues/34656
2. https://github.com/facebook/react-native/issues/34673

#### List of qualified picks

1. https://github.com/facebook/react-native/commit/4b9382c250261aab89b271618f8b68083ba01785
2. https://github.com/facebook/react-native/commit/a5622165c198ac6e7ffff5883d4f269a2c974f2e
3. https://github.com/facebook/react-native/commit/361b310bcc8dddbff42cf63495649291c894d661
4. https://github.com/facebook/react-native/commit/0ce4ea2bf59c9457d6b6c4a18c65bcb561e7853b
5. https://github.com/facebook/react-native/commit/9ecd203eec97e7d21d10311d950c9f8f30c7a4b1
6. https://github.com/facebook/react-native/commit/0cae4959b750ea051dcd04e4c9374e02b1de6e7a
7. https://github.com/facebook/react-native/commit/2d5db284b061aec33af671b25065632e20217f62
8. https://github.com/facebook/react-native/commit/3afef3c16702cefa5115b059a08741fba255b2db
9. https://github.com/facebook/react-native/commit/5451cd48bd0166ba70d516e3a11c6786bc22171a
10. https://github.com/facebook/react-native/commit/82ceb8c3e34560340044ef864ec2fda674b1ba8e
11. https://github.com/facebook/react-native/commit/32d12e89f864a106433c8e54c10691d7876333ee
12. https://github.com/facebook/react-native/commit/81564c1a3dae4222858de2a9a34089097f665e82
13. https://github.com/facebook/react-native/commit/33d1291e1a96497a4f994e9d622248a745ee1ea6

(on hermes)
* https://github.com/facebook/hermes/commit/42fe3572f86c2d8d9648cb5e8414b487cb45931f
* https://github.com/facebook/hermes/commit/211c2d1ed

#### Local commits to backport to main

1.

**Comment by cipolleschi** [comment]
A few commits should be cherry-picked for the codegen:

* https://github.com/facebook/react-native/commit/9ecd203eec97e7d21d10311d950c9f8f30c7a4b1
* https://github.com/facebook/react-native/commit/33d1291e1a96497a4f994e9d622248a745ee1ea6
* https://github.com/facebook/react-native/commit/0ce4ea2bf59c9457d6b6c4a18c65bcb561e7853b
* https://github.com/facebook/react-native/commit/0f0d52067cb89fdb39a99021f0745282ce087fc2

**Reply by dmytrorykun** [reply]
https://github.com/facebook/react-native/commit/0f0d52067cb89fdb39a99021f0745282ce087fc2 is already in 0.70 as https://github.com/facebook/react-native/commit/7a014449d6fb0f907e516ad25d02f37721bed040

**Comment by tido64** [comment]
A reminder to pick this Yoga crash fix: https://github.com/facebook/react-native/commit/361b310bcc8dddbff42cf63495649291c894d661

**Comment by mk0116** [comment]
https://reactnative.dev/docs/new-architecture-app-modules-android
This doc is outdated. It looks like it's for 0.69, not for 0.70.
For example "ndkBuild" changed to "cmkae" in 0.70

**Reply by cortinico** [reply]
Thanks for the report. That slipped through and should be updated (I'll look into it).
However, please note that `ndkBuild{}` is still usable also on 0.70. `cmake{}` is the new recommended way.

**Reply by cortinico** [reply]
Fixed here: https://github.com/facebook/react-native-website/pull/3304

**Comment by cortinico** [comment]
This to potentially pick for the Gradle Plugin:
- https://github.com/facebook/react-native/commit/1a9fb6cb682aa5ff83462e1e2869eb99f3b873fd

**Reply by dmytrorykun** [reply]
This one is already in 0.70 https://github.com/facebook/react-native/commit/0fd776d88906d9ba4c0665864956e389b2ff8ef2

**Reply by cortinico** [reply]
Great then no bump of Gradle Plugin is needed 👍

**Comment by satya164** [comment]
It would be good to cherry-pick this for supporting codegen enabled libraries on iOS in monorepos and for library development: https://github.com/facebook/react-native/commit/a5622165c198ac6e7ffff5883d4f269a2c974f2e

**Comment by mxmzb** [comment]
this one would be sick https://github.com/facebook/react-native/commit/4b9382c250261aab89b271618f8b68083ba01785 🚀

**Comment by michalzuk** [comment]
`Intl` doesn't work properly on iOS without this change 👇 
https://github.com/facebook/hermes/commit/42fe3572f86c2d8d9648cb5e8414b487cb45931f

**Reply by cortinico** [reply]
You're right. We're most likely going to pick both:
- https://github.com/facebook/hermes/commit/42fe3572f
- https://github.com/facebook/hermes/commit/211c2d1ed
And include it inside a 0.70.1 in the next days

**Reply by kelset** [reply]
so we need a new Hermes release?

**Reply by cortinico** [reply]
Yes we need a new tag on Hermes' ends

**Comment by Kudo** [comment]
- https://github.com/facebook/react-native/commit/81564c1a3dae4222858de2a9a34089097f665e82 for hermes profiler
- https://github.com/facebook/react-native/commit/32d12e89f864a106433c8e54c10691d7876333ee for expo-dev-client
- https://github.com/facebook/react-native/commit/82ceb8c3e34560340044ef864ec2fda674b1ba8e for expo integration

thanks very much!

**Reply by dmytrorykun** [reply]
I picked https://github.com/facebook/react-native/commit/32d12e89f864a106433c8e54c10691d7876333ee, but I had to resolve non-trivial merge conflict. Please check that desired behaviour is still there.

**Reply by Kudo** [reply]
i test https://github.com/facebook/react-native/commit/32d12e89f864a106433c8e54c10691d7876333ee on latest 0.70-stable branch and it works like a charm. thanks @dmytrorykun !

**Comment by jamsch** [comment]
This commit fixes a startup crash for Android devices that have no WebView installed. I spotted this crash myself after seeing that Google Play runs a few tests on app submissions, including on devices with no webviews.

https://github.com/facebook/react-native/commit/5451cd48bd0166ba70d516e3a11c6786bc22171a

**Comment by kelset** [comment]
also: https://github.com/facebook/react-native/commit/3afef3c16702cefa5115b059a08741fba255b2db

**Comment by mattmcdev** [comment]
Any chance this fix for the `borderColor` prop not supporting `PlatformColor` on Android can be included?

https://github.com/facebook/react-native/commit/2d5db284b061aec33af671b25065632e20217f62

**Comment by ray-holland-es** [comment]
Would really like this: 
Fix invalid range "t-5" in string transliteration [iOS]
https://github.com/facebook/react-native/commit/0cae4959b750ea051dcd04e4c9374e02b1de6e7a

**Comment by billnbell** [comment]
These all look good..! 0.70.1 :)
